src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml

```diff
@@ -48,8 +48,21 @@
             Text="{Binding Text}" />
       </DataTemplate>
 
+      <DataTemplate DataType="{x:Type nuget:LicenseFileText}">
+        <TextBlock
+            Margin="8,8,0,0"
+            AutomationProperties.LabeledBy="{Binding ElementName=_packageLicenseText}"
```

Comment:
💯

Reply:
Make sure to test this works as expected using narrator and windows inspect object

Reply:
I could use some help testing the narrator.